frontend/src/app/Settings/ManageOrganization.test.tsx

```diff
@@ -7,7 +7,6 @@ import {
   waitForElementToBeRemoved,
 } from "@testing-library/react";
 import { Provider } from "react-redux";
-import { DeepPartial } from "redux";
```

Comment:
`DeepPartial` is no longer exported by redux but I checked what it was actually importing on the main branch and it's just that type definition below so I copied and pasted it.